deployment/dynatrace-azure-logs.sh

```diff
@@ -338,6 +348,12 @@ else
         read -p "Enter EventHub name: " EVENT_HUB_NAME
     done
     echo ""
+
+    echo "Please provide filter config in key-value pair format for example: FILTER.GLOBAL.MIN_LOG_LEVEL=Warning"
+    while ! [[ "${FILTER_CONFIG}" =~ $FILTER_CONFIG_REGEX ]]; do
```

Comment:
FILTER_CONFIG should be optional - with thic code it became required?

Reply:
Thanks for catching that - I made it optional.